pkg/library/libraryapplyconfiguration/operator_launch_helpers.go

```diff
@@ -42,11 +47,24 @@ func (a SimpleOperatorStarter) RunOnce(ctx context.Context) error {
 		informer.WaitForCacheSync(ctx)
 	}
 
-	errs := []error{}
+	knownControllersSet := sets.NewString()
+	for _, controllerRunner := range a.ControllerNamedRunOnceFns {
+		knownControllersSet.Insert(controllerRunner.ControllerInstanceName())
```

Comment:
add an error for every duplicate named controller to prevent someone from using the same name twice.

Reply:
done